docs/index.rst

```diff
@@ -208,6 +208,39 @@ steps:
          Creating test case test_login_1 for requirement: Login Example.
          Linking test case test_login_1 to requirement: Login Example.
 
+test-plan command
++++++++++++++++++
+
+The test-plan command allows to create either parent or child test plans. This
+is accomplished by passing or not the ``--parent-name`` option, if it is
+present then a child test plan will be created and linked with the specified
+parent and, if not specified, a parent test plan will be created.
+
+The test plan ID and the test plan parent's ID will be generated after their
+names by replacing some special characters and converting spaces to ``_``. You
+don't need to worry about the ID since Betelgeuse will take care of that for
+you, just make sure to pass the exact name for both the test plan and parent
+test plan in order to find the expected work items on Polarion.
+
+The examples below show how to create a parent test plan and a test plan:
+
+.. code-block:: console
+
+    $ betelgeuse test-plan --name "Test Plan Parent Name" PROJECT_CLOUD
+    Created new Test Plan Test Plan Parent Name with ID Test_Plan_Parent_Name.
+
+    $ betelgeuse test-plan \
+        --name "Test Plan Name" \
+        --parent-name "Test Plan Parent Name" \
+        PROJECT_CLOUD
+    Created new Test Plan Test Plan Name with ID Test_Plan_Name.
+
+.. note::
+
+    test-plan command accepts ``--plan-type`` as an option and it can be set as
+    ``release`` or ``iteration``. If the option is not specified the default
+    value is ``release``.
```

Comment:
How about:

    Use ``--plan-type`` to set the plan type of a test plan to ``release`` or ``iteration``.  The default value is ``release``.